.gitmodules

```diff
@@ -13,3 +13,6 @@
 [submodule "homeassistant/components/frontend/www_static/polymer/home-assistant-js"]
 	path = homeassistant/components/frontend/www_static/polymer/home-assistant-js
 	url = https://github.com/balloob/home-assistant-js.git
+[submodule "homeassistant/external/vera"]
+	path = homeassistant/external/vera
+	url = https://github.com/jamespcole/home-assistant-vera-api.git
```

Comment:
_Disclaimer: you can ignore this comment if you think this is too nit picky_

If you would rename your repo to `vera-api-python` it will be easier for people to find. Then more people will use it and they might end up contributing code to it.